maven-embedder/src/main/java/org/apache/maven/cli/CLIManager.java

```diff
@@ -167,7 +170,7 @@ public void displayHelp( PrintStream stdout )
 
         HelpFormatter formatter = new HelpFormatter();
 
-        formatter.printHelp( pw, HelpFormatter.DEFAULT_WIDTH, "mvn [options] [<goal(s)>] [<phase(s)>]", "\nOptions:",
+        formatter.printHelp( pw, 120, "mvn [options] [<goal(s)>] [<phase(s)>]", "\nOptions:",
```

Comment:
Why is this now a literal and not a static final? Ideally, we could calculate the width of the terminal, but this is out of scope here.

Reply:
+1, I see no reason why to replace this. I would expect SonarQube to complain about magic numbers.